packages/did/src/DidDetails/LightDidDetails.utils.ts

```diff
@@ -107,3 +155,14 @@ export function decodeAndDeserializeAdditionalLightDidDetails(
     serviceEndpoints: deserialized[SERVICES_KEY_MAP_KEY],
   }
 }
+
+export function getDefaultMigrationHandler(
```

Comment:
Q: Should this maybe be moved into the integrationtest utils? Seems to be only used there and I wouldn't expose a function which does resigning per default to the user of the sdk

Reply:
Makes sense -> https://github.com/KILTprotocol/sdk-js/pull/447/commits/d669855f5582a3e97384bf86fe8d1a1240876a66